go/stellar/stellarsvc/frontend_test.go

```diff
@@ -1033,12 +1033,14 @@ func TestGetPaymentsLocal(t *testing.T) {
 	require.Equal(t, "", p.ToUsername)
 	require.Equal(t, "", p.ToAssertion)
 
+	err = tcs[1].Srv.walletState.RefreshAll(tcs[1].MetaContext(), "test")
+	require.NoError(t, err)
```

Comment:
was this added because there's a new race that could affect real apps?

Reply:
I fixed it in ImportSecretKey (and removed these lines)